lib/alephant/lookup/lookup_cache.rb

```diff
@@ -0,0 +1,67 @@
+require 'dalli-elasticache'
+require 'alephant/logger'
+
+module Alephant
+  module Lookup
+    class LookupCache
+      include Logger
+
+      DEFAULT_TTL  = 5
+
+      def initialize(config)
+        @config = config || {}
```

Comment:
@samfrench I would add an `attr_reader :config` and then modify the latter calls to `@config` to just `config`. Reads a bit nicer